### DISCUSSION TITLE
Quit shilling github

### DISCUSSION THREAD

**Discussion by Kobaxidze256** [git-lfs/git-lfs#5909] [General] [answered]
I opened https://git-lfs.com for the first time just now. I do not like this sentence
```
There is no step three. Just commit and push to GitHub as you normally would; for instance, if your current branch is named main:
```
github is not FOSS, neither does it have any humanitarian obligations (see copilot GPL violations) and neither is it the best Git Forge (https://codeberg.org takes the cake)

**Comment by Kobaxidze256** [comment]
I just saw the license of this project... L

**Comment by Kobaxidze256** [comment]
I just saw the language the project is written in...

**Comment by chrisd8088** [comment, marked as answer]
Hey, thanks for your comments, and I'm sorry our project doesn't meet your expectations.

We try to be neutral in our documentation and replies to users and not recommend any particular Git hosting service over another.

The [mention](https://github.com/git-lfs/git-lfs.github.com/blob/50756783840a2df4e521610b78362954b8fb4a8b/_includes/home/secondary.html#L32) of GitHub in our home page is left over from a year or two ago when we moved the page to its current `git-lfs.com` domain name from `git-lfs.github.com`, which was the home page's domain name for many years; we just didn't notice the legacy text at the time.

Anyone can create a PR in the `git-lfs/git-lfs.github.com` [repository](https://github.com/git-lfs/git-lfs.github.com/) to adjust the wording, and if no one does so over the next while, I'll try to remember to do so myself.  (Note that the repository's name reflects the original domain name, but any changes we merge will be published to the `git-lfs.com` domain.)

The [specifications](https://github.com/git-lfs/git-lfs/tree/9d69005f1be8d860f4f5ee51dd2a81758ea1c1cd/docs/api) of the Git LFS APIs are public and so there are multiple alternative implementations of the client and server sides of the APIs.  There's a [list](https://github.com/git-lfs/git-lfs/wiki/Implementations) of these on our wiki.  If this project's license or code don't meet your needs, you might want to look at the other choices available, or write a new one, of course!  (If you pursue the latter option, please go ahead and add your new implementation to the list on the wiki!)

For some historical background as to why this project is affiliated closely with GitHub, this now-archived [blog post](https://web.archive.org/web/20240302022611/https://blog.developer.atlassian.com/contributing-to-git-lfs/) from an Atlassian developer describes how Git LFS emerged from the joint efforts of teams at GitHub and Atlassian, as well as other companies like GitLab.  There's also a Linux Foundation [video](https://www.youtube.com/watch?v=p3Pse1UkEhI) whose introductory section covers some of the same history.  The Git LOB project they discuss is still [available](https://github.com/atlassian/git-lob), although no longer under development, obviously.

**Reply by chrisd8088** [reply]
I've updated the project home page in git-lfs/git-lfs.github.com@9e728c00b57c7bbeaa69288c8cc125c1c114c496.

**Reply by Kobaxidze256** [reply]
WOW, thanks. I appreciate it a lot!
